.github/workflows/ci.yml

```diff
@@ -234,7 +232,7 @@ jobs:
       platform: linux/${{ matrix.platform }}
       platform_base_image: ${{ fromJson(needs.metadata.outputs.platform_config)[format('{0}', matrix.platform)].minimal_base_image }}
       build_cache:  ${{ fromJson(needs.config.outputs.json).build_cache[format('{0}-installer', matrix.platform)] }}
-      toolchain: gcc11
+      toolchain: llvm
```

Comment:
The build caches are separate for different toolchains and platforms. I.e. yes, this will create new caches, but the old ones are either deleted automatically after a while or we can go delete them manually. The PR removes the llvm devdeps build (being superceded by the assets), and thus reduces the overall number of caches (until we add an additional build for CUDA 12).